apiary.apib

```diff
@@ -216,9 +297,9 @@ or set account properties or destroy their accounts.
 
 Destroying an account, and changing account properties
 like username and password, require to pass the user's
-password as `x-password` header.
+bearer token as `Authorization` header.
 
-### Sign Up [PUT /session/account?include=session]
+### Sign Up [PUT /session/account]
```

Comment:
Unfortunately `?include=session` doesn't work here, because as far as I understand the session is not a relation of the account, but the other way around (that is we can ask to include the `account` when requesting a `session` but not the `session` when requesting the `account` with `/session/account`).

If I understand correctly, the goal here is to give back the session token for the client to store it for further requests. Can we send back that token as part of the header?